src/ui/components/Rating/index.js

```diff
@@ -66,18 +71,35 @@ export default class Rating extends React.Component {
   }
 
   render() {
-    const { readOnly, size } = this.props;
+    const { i18n, rating, readOnly, size } = this.props;
     const cls = classNames('Rating', {
       'Rating--editable': !readOnly,
       'Rating--small': size === 'small',
     });
+    let description;
+    if (rating) {
+      description = i18n.sprintf(
+        // L10n: This is for showing the star rating of an add-on.
+        i18n.ngettext('Rated %(starCount)s star',
```

Comment:
Might be better to right `Rated %(starCount)s out of 5`; visually it's clear the rating is out of five but a screenreader saying "it's 4 stars" won't give much context (eg "4 out of what? 4 out of 5? Out of 10?")

The "stars" bit doesn't even add much either, so I think a plain number here (X of out 5) would work better; bonus it's less complex code.